src/input/SimModSuite.h

```diff
@@ -456,6 +459,27 @@ class SimModSuite : public MeshInput {
     // Set mesh size on vertices, edges, surfaces and regions
     setMeshSize(model, meshCase, MeshAtt);
 
+    if (MeshAtt.useVelocityAwareMeshing) {
+      logInfo() << "Enabling velocity aware meshing";
+      easiMeshSize =
+          EasiMeshSize(MeshAtt.easiFileName, MeshAtt.targetedFrequency,
+                       MeshAtt.elementsPerWaveLength, model, groupMap);
+      auto easiMeshSizeFunc = [](pSizeAttData sadata, void *userdata){
+        auto* easiMeshSize = static_cast<EasiMeshSize*>(userdata);
+        std::array<double, 3> pt{};
+        int haspt = SizeAttData_point(sadata, pt.data());
+        if (!haspt) {
+          logError() << "!haspt";
+        }
+        return easiMeshSize->getMeshSize(pt);
+        };
+      // set the user-defined function for anisotropic size
+      MS_setSizeAttFunc(meshCase, "setCustomMeshSize", easiMeshSizeFunc, &easiMeshSize);
+      // Relative anisotropic size is set for the entire model through the
+      // function anisoSize
+      MS_setMeshSize(meshCase, GM_domain(model), MS_userDefinedType | 2, 0, "setCustomMeshSize");
```

Comment:
We need type=1:
```
type = 1 | MS_userDefinedType for absolute, type = 2 | MS_userDefinedType for relative user defined size).```

Reply:
I must add that I tested it on a real case study and it did not generate a mesh after 30min, probably because of this type 2.